src/Tribe/Admin/Help_Page.php

```diff
@@ -74,6 +115,9 @@ public function load_assets() {
 			null,
 			'admin_enqueue_scripts'
 		);
+
+		// Add the built-in accordion.
+		wp_enqueue_script( 'jquery-ui-accordion' );
```

Comment:
This is too early to enqueue scripts. `load_assets` is called with `tribe_common_loaded`, we might want to attach it to a later moment.

Reply:
@bordoni  Actually it is called from this hook.
https://github.com/the-events-calendar/tribe-common/pull/2226/files#diff-af4649c5fe351226b2ecc6d47af75739887c092152f15e8f43135dc2fdbfab3fL55